app/views/errors/no_patient_with_draft_vaccination_record.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= h1 "Error: page not available", size: "xl" %>
+
+<p class="nhsuk-body">
+  You do not have a vaccination record open, or it has timed out.
+</p>
+
+<p class="nhsuk-body">
+  To see a vaccination record, you can search for it in Mavis in either:
```

Comment:
We don't typically use the service name ("Mavis") in the body content. I think it would be good to get some input from the designs on this page.

I'm also wondering if we need that much content on this page and if one of our standard error pages will do?